### ISSUE TITLE
How to apply dropout for getting prediction from the last layer of a trained model?

### ISSUE THREAD

**Issue by umairalipathan** [tensorflow/tensorflow#8398]
I have used transfer learning to retrain/fine-tune Inception-V3 model for image classification on my own data in Tensorflow. However, when I test the image classification after training, I get the following error:

> "Invalid argument: You must feed a value for placeholder tensor 'final_layer/dropout/Placeholder' with dtype float"

I am using the following code for getting the predictions from the final layer:

`softmax_tensor = sess.graph.get_tensor_by_name('final_layer/final_result/Softmax:0')
			predictions = sess.run(softmax_tensor, {'DecodeJpeg/contents:0': image_data})`

I guess the error is caused by not applying the dropout placeholder in the second statement, as the training was performed with dropout layer. How do I add a dropout placeholder here?

**Comment by concretevitamin** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!

**Comment by concretevitamin** [comment]
(Without looking at the code, I am guessing you can try
```predictions = sess.run(softmax_tensor, 
         {'DecodeJpeg/contents:0': image_data, 'final_layer/dropout/Placeholder': 0.0})
```
or something like that.  Regardless, please ask usage questions on Stackoverflow; thanks!)
